### PR TITLE
Satellite 6.5 changes

### PR DESCRIPTION
**Pull request by ogajduse**
New Satellite 6.5 release is out. Changes in this PR are reflecting the default state of the answer file that comes with Sat 6.5.
I tested these changes with playbook similar to the one that is in Readme. Works as expected.

### CODE REVIEW

**Comment by ogajduse** [comment]
May I ask you for a review? @greg-hellings @seandst @st-murray

File: templates/satellite-answers.yaml [hunk 1]
```diff
@@ -24,18 +24,18 @@ foreman:
   custom_repo: true
   configure_epel_repo: false
   configure_scl_repo: false
-  server_ssl_cert: /etc/pki/katello/certs/katello-apache.crt
-  server_ssl_key: /etc/pki/katello/private/katello-apache.key
-  server_ssl_ca: /etc/pki/katello/certs/katello-default-ca.crt
-  server_ssl_chain: /etc/pki/katello/certs/katello-server-ca.crt
+  server_ssl_cert: "/etc/pki/katello/certs/katello-apache.crt"
```
Comment (greg-hellings) [inline templates/satellite-answers.yaml:27]: The quotation marks *should* be unnecessary. But also not a problem.

**Review by greg-hellings — APPROVED** [review]
Sorry for the delay, I've been on PTO and the rest of the team has been focused on a big P0 project. Looks good to me.  I tagged the current master as 6.4-1.0.0 and will merge this post haste.

File: templates/satellite-answers.yaml (hunk 1 above)
Comment (ogajduse) [reply templates/satellite-answers.yaml:27]: I know about that. I just wanted to keep a track with the Satellite's original answer file.

**Comment by ogajduse** [comment]
Thanks for the quick merge. Can you also tag the 6.5 scenario, please?

**Comment by greg-hellings** [comment]
6.5-1.0.0 tagged.

**Comment by ogajduse** [comment]
@greg-hellings Thanks!

**Pull request merged**